docs/architecture/adr/index.mdx

```diff
@@ -23,9 +23,16 @@ The AD will also serve as a base for proposing and planning technical debt.
 
```

Comment:
Could we add some text to the above section explaining that an ADR is expected to update the architecture documentation to reflect the latest status?